### PR TITLE
Implemented filter distribution functionality

### PR DESCRIPTION
**Pull request by deveshidwivedi**
This pull request adds distribution filtering to the `SearchResults` component, allowing users to filter search results based on child distributions.
Currently, only a few distributions are being fetched.

### CODE REVIEW

**Comment by Apurv428** [comment]
Some more changes
1. Add margin between navbar and faq box
2. Remove blog and contact from navbar

**Comment by deveshidwivedi** [comment]
> Some more changes
> 
> 1. Add margin between navbar and faq box
> 2. Remove blog and contact from navbar

Updated

File: react-frontend/src/components/SearchResults.jsx [hunk 1]
```diff
@@ -7,26 +7,52 @@ function SearchResults({ results = [], showDesc, itemsPerPage, searchPerformed }
...
+    setDistributions(uniqueDistributions);
+  }, [results]);
+
+  const getDistributions = () => {
+    if (results.length === 0) return [];
+
+    // Log the first item to understand its structure
+    console.log('First result item:', results[0]);
+
+    const distributionSet = new Set(results.map(result => result.ostag).filter(Boolean));
```
[the remark this answers is not in the document]
Comment (deveshidwivedi) [reply react-frontend/src/components/SearchResults.jsx]: Updated to fetch and display all the distributions, however, results are seen only for some of them

**Comment by Apurv428** [comment]
> Collaborator Author

Please add margin between faq and footer too.
Sorry forgot to mention initially.

File: react-frontend/src/components/SearchResults.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (deveshidwivedi) [reply react-frontend/src/components/SearchResults.jsx]: Added

**Comment by rachejazz** [comment]
Since this PR is not merging to master, I'm approving it anyway.

**Pull request merged**